pkg/logger/logger.go

```diff
@@ -218,10 +249,39 @@ func (rl *ReceptorLogger) Log(level int, format string, v ...interface{}) {
 
 	if logLevel >= level {
 		rl.Logger.SetPrefix(prefix)
+		format, v = rl.appendSuffix(format, v)
 		rl.Logger.Printf(format, v...)
 	}
 }
 
+func (rl *ReceptorLogger) appendSuffix(format string, v []interface{}) (string, []interface{}) {
```

Comment:
Should we change to `string` from `interface{}` here too?

```suggestion
func (rl *ReceptorLogger) appendSuffix(format string, v []string) (string, []string) {
```

Reply:
I see this was marked resolved but the question wasn't addressed. Can you please help me understand?

Reply:
github auto-resolved when I committed the suggestion. I reverted the commit and manually unresolved just now. Explanation below:

Existing loggers use something like this: `func (rl *ReceptorLogger) Log(level int, format string, v ...interface{}) ` to handle formatting like "%s" "%i" etc... so private 

`func (rl *ReceptorLogger) appendSuffix(format string, v []interface{}) (string, []interface{}) {` 

needs to as well.

Reply:
Thanks for the explanation!